topdown/strings.go

```diff
@@ -384,6 +384,25 @@ func builtinSprintf(a, b ast.Value) (ast.Value, error) {
 	return ast.String(fmt.Sprintf(string(s), args...)), nil
 }
 
+func builtinRepeat(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	s, err := builtins.StringOperand(operands[0].Value, 1)
+	if err != nil {
+		return err
+	}
+
+	count, err := builtins.IntOperand(operands[1].Value, 2)
+	if err != nil {
+		return err
+	}
+
+	if count < 0 {
+		return builtins.NewOperandErr(2, "count must be greater than zero")
```

Comment:
💭 what's up with `repeat("hi", 0)`? It seems like it's valid, but it's also not _greater than zero_ 😉

Reply:
Gonna make it strictly greater than zero,  `repeat("hi", 0)` seems like a weird way of defining an empty string.